<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLayer</class>
 <widget class="QWidget" name="GameLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="UnderBarColLeft">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>51</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 10%;
border: 3.5px solid black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UnderBarColRight">
   <property name="geometry">
    <rect>
     <x>1079</x>
     <y>390</y>
     <width>51</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity: 10%;
border: 3.5px solid black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UnderBarCenterVeryBottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>701</y>
     <width>1280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 78, 78);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BarTop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>1280</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/wood.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="BarTopBottomLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>1280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(54, 54, 54);
border-bottom: 3px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_VodkaButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>580</y>
     <width>122</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/vodka);
}

QPushButton::hover {
image: url(:/images/liquor/vodka_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>147</width>
     <height>162</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="TopColLeft">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>21</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TopColCentre">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>981</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="TopColRight">
   <property name="geometry">
    <rect>
     <x>1129</x>
     <y>0</y>
     <width>21</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="d_RumButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>580</y>
     <width>75</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/rum);
}

QPushButton::hover {
image: url(:/images/liquor/rum_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_BurbonButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>125</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/burbon);
}

QPushButton::hover {
image: url(:/images/liquor/burbon_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_CollinsGlassButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>270</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/glasses/collins.png);
}

QPushButton::hover {
image: url(:/images/glasses/collins_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DrinksPanel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>1081</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
border-bottom: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="d_RocksGlassButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>300</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/glasses/rocks.png);
}

QPushButton::hover {
image: url(:/images/glasses/rocks_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_LimeWedgeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/garnish/lime_wedges);
}

QPushButton::hover {
image: url(:/images/garnish/lime_wedges_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_OliveButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/garnish/olives);
}

QPushButton::hover {
image: url(:/images/garnish/olives_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_OrangeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/garnish/orange_peels);
}

QPushButton::hover {
image: url(:/images/garnish/orange_peels_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DrinkMakeArea">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>231</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/steel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Background">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>-20</y>
     <width>1479</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="d_DrinkAlgoBox">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(250, 255, 155);*/
background-color: rgb(250, 255, 155);
border: 2px dotted black;
color: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_MartiniGlassButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Martini</string>
   </property>
  </widget>
  <widget class="QPushButton" name="d_GinButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>570</y>
     <width>77</width>
     <height>133</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/gin);
}

QPushButton::hover {
image: url(:/images/liquor/gin_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_KahluaButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>81</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Kahlua</string>
   </property>
   <property name="toolTipDuration">
    <number>5</number>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/kahlua.png);
}

QPushButton::hover {
image: url(:/images/liquor/kahlua_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_OrangeLiquorButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>570</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/ingredients/tripleSec.png);
}

QPushButton::hover {
image: url(:/images/ingredients/tripleSec_Hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_CopperMugButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>340</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/glasses/CopperMug.png);
}

QPushButton::hover {
image: url(:/images/glasses/CopperMug_Hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_BittersButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/ingredients/bitters);
}

QPushButton::hover {
image: url(:/images/ingredients/bitters_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_ShakertinButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/tools/shakertin);
}

QPushButton::hover {
image: url(:/images/tools/shakertin_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="d_TrashLabel">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>580</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/tools/trash.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UnderBarCenter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>881</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(78, 78, 78);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UnderBarCenterBottom">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>881</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 38, 38);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UnderBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>1280</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0.5);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gamegraphics.qrc">:/images/bar/under_bar.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="d_IceButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>550</y>
     <width>225</width>
     <height>225</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/tools/icewell);
}

QPushButton::hover {
image: url(:/images/tools/icewell_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_ServeDrinkButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Serve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="d_StirButton">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>510</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SPOON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="d_LimeJuiceButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/ingredients/LimeJuice.png);
}

QPushButton::hover {
image: url(:/images/ingredients/LimeJuice_Hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_GingerBeerButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/ingredients/GingerBeer.png);
}

QPushButton::hover {
image: url(:/images/ingredients/GingerBeer_Hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_SinkButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>225</width>
     <height>225</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/bar/empty_icewell);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_TequilaButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>122</width>
     <height>125</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/liquor/tequila);
}

QPushButton::hover {
image: url(:/images/liquor/tequila_hover);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="triqa_pause">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>114</width>
     <height>94</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="menugraphics.qrc">:/images/menu/triqa_pause.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PauseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>18</y>
     <width>54</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/menu/pause);
}

QPushButton::hover {
image: url(:/images/menu/pause_hover);
border-image: url(:/images/menu/pause_hover) 1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="d_RocksGlassButtonSpawn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: transparent;
border: none;
image: url(:/images/glasses/rocks.png);
}

QPushButton::hover {
image: url(:/images/glasses/rocks_hover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DrinkViewport" name="v_DrinkViewport">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>141</y>
     <width>256</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px;
background-color: transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Background_2">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>-20</y>
     <width>1479</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0,100);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>Background</zorder>
  <zorder>Background_2</zorder>
  <zorder>TopColCentre</zorder>
  <zorder>UnderBar</zorder>
  <zorder>UnderBarCenter</zorder>
  <zorder>UnderBarCenterBottom</zorder>
  <zorder>TopColRight</zorder>
  <zorder>TopColLeft</zorder>
  <zorder>UnderBarColLeft</zorder>
  <zorder>UnderBarColRight</zorder>
  <zorder>UnderBarCenterVeryBottom</zorder>
  <zorder>BarTop</zorder>
  <zorder>BarTopBottomLine</zorder>
  <zorder>d_CollinsGlassButton</zorder>
  <zorder>d_RocksGlassButton</zorder>
  <zorder>d_LimeWedgeButton</zorder>
  <zorder>d_OrangeButton</zorder>
  <zorder>DrinkMakeArea</zorder>
  <zorder>d_DrinkAlgoBox</zorder>
  <zorder>d_MartiniGlassButton</zorder>
  <zorder>d_GinButton</zorder>
  <zorder>d_KahluaButton</zorder>
  <zorder>d_OrangeLiquorButton</zorder>
  <zorder>d_CopperMugButton</zorder>
  <zorder>d_BittersButton</zorder>
  <zorder>d_ShakertinButton</zorder>
  <zorder>d_OliveButton</zorder>
  <zorder>d_TrashLabel</zorder>
  <zorder>d_IceButton</zorder>
  <zorder>d_ServeDrinkButton</zorder>
  <zorder>d_StirButton</zorder>
  <zorder>d_LimeJuiceButton</zorder>
  <zorder>d_GingerBeerButton</zorder>
  <zorder>d_SinkButton</zorder>
  <zorder>d_TequilaButton</zorder>
  <zorder>d_VodkaButton</zorder>
  <zorder>d_RumButton</zorder>
  <zorder>d_BurbonButton</zorder>
  <zorder>DrinksPanel</zorder>
  <zorder>triqa_pause</zorder>
  <zorder>PauseButton</zorder>
  <zorder>v_DrinkViewport</zorder>
  <zorder>d_RocksGlassButtonSpawn</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrinkViewport</class>
   <extends>QGraphicsView</extends>
   <header>drinkviewport.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gamegraphics.qrc"/>
  <include location="menugraphics.qrc"/>
 </resources>
 <connections/>
</ui>
